tools/slurm/README.md

```diff
@@ -5,7 +5,7 @@
 1. Install Terraform following [these instructions](https://developer.hashicorp.com/terraform/downloads).
 2. Download the [GCP CLI](https://cloud.google.com/sdk/docs/install-sdk) and run `gcloud auth application-default login` to get credentials.
 3. Run `make slurmcluster` from the root of the repo and wait (up to 10 minutes) for it to start.
-   - To specify which container runtime environment to use, pass in `FLAGS="-c {container_run_type}"` to `make slurmcluster`. Choose from either `singularity` (default), `podman`, or `enroot`.
+   - To specify which container runtime environment to use, pass in `FLAGS="-c {container_run_type}"` to `make slurmcluster`. Choose from either `singularity` (default), `podman`, or `enroot`. If utilizing Slurmcluster with Determined Agents, `docker` container runtime environment is the sole available option.
```

Comment:
I don't think we support Slurmcluster with docker

Reply:
For clarity, the intent is to no longer give customers the option to use slurm with determined agents.

At least, that's what I understood from slack thread: https://hpe-aiatscale.slack.com/archives/C06GMG83ZE0/p1718224380888699

Reply:
Corrected it. Let me know mentioning it straight out makes sense or not.

Reply:
I think we want to remove the entire last sentence, `If utilizing Slurmcluster with Determined Agents, `docker` container runtime environment is the sole available option.`

We don't want customers to use Slurmcluster with Determiend Agents. At least, that's my understanding.